Editor/Resources/EditorWindow/GameLiftPlugin.cs

```diff
@@ -0,0 +1,204 @@
+        public AwsCredentialsCreation CreationModel;
+        public AwsCredentialsUpdate UpdateModel;
+
+        private static GameLiftPlugin GetWindow()
+        {
+            var inspectorType = Type.GetType("UnityEditor.GameView,UnityEditor.dll");
+            var window = GetWindow<GameLiftPlugin>(inspectorType);
+            window.titleContent = new GUIContent("Amazon GameLift", window.icon);
+            return window;
+        }
+
+        [MenuItem("Amazon GameLift/Show Amazon GameLift Window", priority = 0)]
+        public static void ShowWindow()
+        {
+            GetWindow();
+        }
+
+        [MenuItem("Amazon GameLift/Bring Panel to Front", priority = 1)]
+        public static void FocusPanel()
+        {
+            ShowWindow();
+        }
+
+        [MenuItem("Amazon GameLift/Set AWS Account Profiles", priority = 100)]
+        public static void OpenAccountProfilesTab()
+        {
+            GetWindow().OpenTab("Credentials");
+        }
+
+        [MenuItem("Amazon GameLift/Host with Anywhere", priority = 101)]
+        public static void OpenAnywhereTab()
+        {
+            GetWindow().OpenTab("Anywhere");
+        }
+
+        [MenuItem("Amazon GameLift/Host with Managed EC2", priority = 102)]
+        public static void OpenEC2Tab()
+        {
+            GetWindow().OpenTab("EC2");
+        }
+
+        [MenuItem("Amazon GameLift/Import Sample Game", priority = 103)]
+        public static void ImportSampleGame()
+        {
+            string filePackagePath = $"Packages/{Paths.PackageName}/{Paths.SampleGameInPackage}";
+            AssetDatabase.ImportPackage(filePackagePath, interactive: true);
+        }
+
+        [MenuItem("Amazon GameLift/Help/Documentation", priority = 200)]
+        public static void OpenDocumentation()
+        {
+            Application.OpenURL(Urls.AwsHelpGameLiftUnity);
+        }
+
+        [MenuItem("Amazon GameLift/Help/AWS GameTech Forum", priority = 201)]
+        public static void OpenGameTechForums()
+        {
+            Application.OpenURL(Urls.AwsGameTechForums);
+        }
+
+        [MenuItem("Amazon GameLift/Help/Report Issues", priority = 202)]
+        public static void OpenReportIssues()
+        {
+            Application.OpenURL(Urls.GitHubAwsLabs);
+        }
+
+        private void CreateGUI()
+        {
+            _root = rootVisualElement;
+            _mVisualTreeAsset = UnityEngine.Resources.Load<VisualTreeAsset>("EditorWindow/GameLiftPlugin");
+            if (_mVisualTreeAsset == null)
+            {
+                return;
+            }
+
+            VisualElement uxml = _mVisualTreeAsset.Instantiate();
+            _root.Add(uxml);
+            ApplyText();
+
+            CoreApi = CoreApi.SharedInstance;
+            var awsCredentials = AwsCredentialsFactory.Create();
+            CreationModel = awsCredentials.Creation;
+            UpdateModel = awsCredentials.Update;
+            RefreshProfiles();
```

Comment:
Can initialization of non ux elements move elsewhere? maybe a constructor?

```
 CoreApi = CoreApi.SharedInstance;
var awsCredentials = AwsCredentialsFactory.Create();
UpdateModel = awsCredentials.Update;
```

The CreationModel doesn't seem to be used anymore, please remove if that's the case.

Reply:
CreationModel is Used within AwsUserProfilesPage.cs